test/config_parser_SUITE.erl

```diff
@@ -160,6 +174,79 @@ compare_unordered_lists(L1, L2, F) ->
 
 compare_ordered_lists([H1|T1], [H1|T2], F) ->
     compare_ordered_lists(T1, T2, F);
+compare_ordered_lists([{backends, Backends}], [{backends, Backends2}], _) ->
+    Sns = proplists:get_value(sns, Backends, []),
+    Sns2 = proplists:get_value(sns, Backends2, []),
+    Push = proplists:get_value(push, Backends, []),
+    Push2 = proplists:get_value(push, Backends2, []),
+    Http = proplists:get_value(http, Backends, []),
+    Http2 = proplists:get_value(http, Backends2, []),
+    compare_unordered_lists(Sns, Sns2),
+    compare_unordered_lists(Push, Push2),
+    compare_unordered_lists(Http, Http2);
```

Comment:
For me something like this looks cleaner, but perhaps there is an even nicer way of expressing this. This could be reused below.
```suggestion
Keys = [push, http, sns],
       [compare_unordered_lists(proplists:get_value(Key, Backends, []),
                                proplists:get_value(Key, Backends2, [])) 
       || Key <- Keys]
```